eZ/Publish/Core/QueryType/QueryParameterContentViewQueryTypeMapper.php

```diff
@@ -61,13 +61,13 @@ private function extractParameters(ContentView $contentView)
 
     /**
      * @param ContentView $contentView
-     * @param string $queryParameterValue
+     * @param string|string[] $queryParameterValue
      *
      * @return mixed
      */
     private function evaluateExpression(ContentView $contentView, $queryParameterValue)
     {
-        if (substr($queryParameterValue, 0, 2) === '@=') {
+        if (is_string($queryParameterValue) && substr($queryParameterValue, 0, 2) === '@=') {
```

Comment:
You did a great job @nicolas-bastien, thank you.

> I'm assuming this fix is needed for demo bundle as we speak

Close, for now we need it only in ezplatform-com (https://github.com/ezsystems/ezplatform-com/pull/58) but probably it will be also needed in demo soon.